index.html

```diff
@@ -79,13 +79,14 @@ <h3>0</h3>
     </section>
     <section class="hidden pop-ups clue-pop-up">
       <h1 class="clue-text">CLUE TEXT</h1>
-      <h2 class="answer-message-text">CORRECT!</h2>
+      <h2 class="answer-message-text">PLEASE ENTER AN ANSWER</h2>
```

Comment:
Great UI edit.

Reply:
Instead of writing your text in all caps, usually a better convention is to use a css property on that element like `text-transform: uppercase` (I can't actually remember why to be honest, but it might be an accessibility thing.)